securedrop/tests/functional/test_admin_interface.py

```diff
@@ -14,6 +14,10 @@ def test_admin_interface(self):
         self._admin_adds_a_user()
         self._new_user_can_log_in()
         self._admin_can_edit_new_user()
+        self._admin_visits_admin_interface()
```

Comment:
What about having a separate function for this ? Similar to what there is in page-layout

```
    def test_admin_edit_hotp_secret(self):
        self._admin_logs_in()
        self._admin_visits_admin_interface()
        self._admin_adds_a_user()
        self._admin_visits_edit_user()
        self._admin_visits_reset_2fa_hotp()
```